CIP-token-metadata-translations/README.md

```diff
@@ -0,0 +1,153 @@
+                  
+      <information about collection>
+      
+      "strings": {
+              "de-CH": {
+                 <localized information about collection in de-CH>
+              },
+              "it-IT": {
+                 <localized information about collection in it-IT>
+              },
+                    
+              <other languages and cultures>
+      }
+  }
+}
+
+```
+
+This metadata standard extension is backward compatible and it doesn't affect applications using the current standard. Dapps implementing the proposed extended standard can also default on the legacy values if the localized strings are not available on an asset.
+
+> **Note**
+> The metadata's size is a limitation that should be considered as it will eventually push the boundaries of Cardano's transaction limits and scalability. The translations under the "strings" keys can be stored off-chain on an IPFS server. Then, the localized texts will be accessible through an URL, similar to the "image" keys.
+
+### Code example to access localized properties (TypeScript)
+
+To access the localized strings from the fetched metadata for a native asset, we can simply access the JSON properties from the front end by using the user's selected culture:
+
+```
+
+const response = await fetch(`${<BASE_URL>}/policyId/${<policyId>}`);
+            
+const metadata = response.json();
+const policy = metadata["721"][<policy_id>];
+
+function getPolicyString(policy, key, culture="en") {
+    return policy["strings"][culture][key] 
+        ? policy["strings"][culture][key] 
+        : policy[key]; // default value (not localized)
+}
+
+function getAssetString(policy, asset, key, culture="en") {
+    return policy[asset]["strings"][culture][key] 
+        ? policy[asset]["strings"][culture][key] 
+        : policy[asset][key]; // default value (not localized)
+}
+
+console.log(`Default policy property: ${getPolicyString(policy, <policy_property>)}`);
+console.log(`Localized policy property: ${getPolicyString(policy, <policy_property>, "it-IT")}`);
+console.log(`Default asset name: ${getAssetString(policy, <asset_name>, "name")}`);
+console.log(`Localized asset name: ${getAssetString(policy, <asset_name>, "name", "de-CH")}`);
+         
+```
+     
+## Path to Active
+
+### Acceptance Criteria
+- NFT metadata standard extension covers all existing localization needs and use cases on web2.
+- Implementation is compliant with JSON conventions.
+- Implementation is compliant with the [[ISO-639]](https://www.iso.org/standard/4767.html) standard for language code, and the [[ISO-3166]](https://www.iso.org/standard/63545.html) standard for country/region code.
+- Access to localized strings is easy and logical from a coding perspective.
+            
+### Implementation Plan
+Add this new standard to all relevant documentation and references for web3 developers.
+
+NMKR has supported this CIP with peer feedback, and in the future with a pilot implementation of the new standard.

```

Comment:
```suggestion
NMKR has supported this CIP with peer feedback, and in the future with a pilot implementation of this localization method.
```
@granada-pool I don't think we can use the term "standard" here because of the multiple ways this problem might be addressed.  The above is one idea which would also be specific about what would be implemented.

Reply:
I agree! There are indeed several methods to solve this. Although, I really liked the idea of being a bit "aggressive" or "arrogant" and call it a **standard**, just like Ethereum took the liberty of creating the _ERC721 Metadata Standard_ for their NFTs, which has been consolidated by [OpenSea ](https://docs.opensea.io/docs/metadata-standards), and it's even used in Cardano's NFTs. Based on my experience in web2 text localization, this approach is quite solid, and since Cardano would be the first chain ever addressing this language barrier in the space, why not call it a _Cardano standard_? Specially, if NMKR will eventually become larger than OpenSea. 😉

Reply:
Then indeed we could wait until we see other community members publicly calling it a "standard" as was the case with ERC721.  As it is now this is just the most promising _candidate_ for a localisation standard. 😉